ref/appliance-command-reference/cluster.md

```diff
@@ -0,0 +1,27 @@
+The `cluster` command allows management of manage cluster members for the Anbox Cloud Appliance. Using this command, you can join additional nodes to an existing installation or to remove existing nodes.
+
+ See [How to join a machine to the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/29054) for more information.
+
+ ## Usage
+
+    anbox-cloud-appliance cluster <subcommand>
+
+## Subcommands
+
+### `add`
+Adds a new member to the cluster
+
+#### Usage
+
+    anbox-cloud-appliance cluster add <member-name>
```

Comment:
As a user, how can I get the member name? Is this something I should know?

Reply:
You need to choose one for yourselves, taking into account the current number of LXD nodes in the cluster, and name the new one starting from `lxd<existing_number_of_lxd_nodes>.`
https://discourse.ubuntu.com/t/how-to-join-a-machine-to-the-anbox-cloud-appliance/29054